src/api/app/views/webui/users/notifications/index.html.haml

```diff
@@ -7,10 +7,11 @@
     .card.mb-3
       %strong.d-block.d-md-none.p-3{ data: { 'bs-toggle': 'collapse', 'bs-target': '#filters' },
                                   aria: { expanded: true, controls: 'filters' } }
-        Filtered by: #{params[:kind]&.humanize || @filtered_project || 'Unread'}
+        Filtered by: #{@filter_state.map(&:to_s).map(&:humanize).join(", ")} - #{@filter_type.map(&:to_s).map(&:humanize).join(", ")}
```

Comment:
We need to push this into the controller, its too much logic for a view template...